ledger/artifactmanager/artifactmanager.go

```diff
@@ -355,16 +355,16 @@ func (m *LedgerArtifactManager) DeactivateObj(requestRef, objRef record.Referenc
 			HeadRecord: objRef,
 			AmendedRecord: record.Reference{
```

Comment:
Should be just `AmendedRecord: objIndex.LatestStateRef`. No need to create another Record.